apollo-gradle-plugin-incubating/src/test/kotlin/com/apollographql/apollo/gradle/util/TestUtils.kt

```diff
@@ -26,7 +26,9 @@ object TestUtils {
 
     block(dest)
 
-    dest.deleteRecursively()
+    // It's ok to not delete the directory as it will be deleted before next test
+    // During developement, it's easy to keep the testProject around to investigate if something goes wrong
+    // dest.deleteRecursively()
```

Comment:
At Novoda Static Analysis Plugin, we would use jUnit temp folder rule to create the test project into a random folder and then only delete the folder if the test is successful

Reply:
https://github.com/novoda/gradle-static-analysis-plugin/blob/master/plugin/src/test/groovy/com/novoda/test/TestProjectRule.groovy#L59

Reply:
I actually like that the test is always in the same location so that I can use my bash history to replay any commands. I tend to do that a lot with stuff like: 

* `./gradlew -p apollo-gradle-plugin-incubating/build/testProject -Dorg.gradle.debug=true`
* `find apollo-gradle-plugin-incubating/build/testProject/build/generated/sources/apollo `